src/Message/ZipFile.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace App\Message;
+
+/**
+ * The message for Zip file handler.
+ */
+class ZipFile
```

Comment:
The message is very specific, the zip the dataset files, maybe we should call it that?